examples/snuba-lw-deletions/1/snuba-lw-deletions-search-issues.json

```diff
@@ -0,0 +1,13 @@
+{
+  "storage_name": "search_issues",
+  "rows_to_delete": 1000,
+  "conditions": {
+    "project_id": [1],
+    "group_id": [2222222, 3333333, 4444444]
+  },
```

Comment:
with this input format the consumer will be doing the conversion to SQL.

An alternative design would be having the producer do the conversion, which would mean less work for the consumer.

im not sure if one approach should be preferred over the other.

Reply:
> An alternative design would be having the producer do the conversion, which would mean less work for the consumer.

AFAIK, this format is valid only for the search issues and can change if/when we onboard other products on lightweight delete. Whichever approach we choose, let's make sure it remains consistent. I believe if input format changes from one product to another, then we may not have one place (either producer or consumer) to do the conversion. But this is a fair point that needs to be given some thought.

Reply:
> with this input format the consumer will be doing the conversion to SQL.

it does, but formatting the conditions uses the same logic from the api endpoint, so we can re-use that functionality.  it also makes it easier to validate the kafka message. If you had the SQL string as a parameter and you wanted to do that validation in the consumer you'd have to parse it anyway (or at least use a regex)